src/main/kotlin/net/dungeonhub/application/commands/LoggingSystem.kt

```
@@ -409,4 +373,180 @@ class LoggingSystem : Extension() {
             maxValue = 200
         }
     }
+
+    companion object {
+        private val logger = LoggerFactory.getLogger(LoggingSystem::class.java)
+        private lateinit var scheduler: Scheduler
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)